examples/react/file-upload/README.md

```diff
@@ -9,16 +9,15 @@ To run this example:
   cd w3ui
   ```
 
-- Install dependencies and build:
+- Install dependencies:
 
   ```sh
-  npm install
-  npm run build
+  pnpm install
   ```
 
 - Change to this example directory and run the example:
 
   ```sh
   cd examples/react/file-upload
-  npm start
+  pnpm run dev
```

Comment:
ooh let's keep `(p)npm start` working. It could be an alias for `dev` so both work.